python/lsst/ts/observatory/control/remote_group.py

```diff
@@ -724,6 +724,20 @@ async def standby(self):
 
         await self.set_state(salobj.State.STANDBY)
 
+    def set_rem_loglevel(self, level):
+        """Set remotes log level.
+
+        Usefull to prevent the internal salobj warnings when read queues are
+        filled up.
```

Comment:
filling up, no? it warns before full I thought...

Reply:
yes